wifiphisher/pywifiphisher.py

```diff
@@ -758,8 +763,7 @@ def start(self):
         if not args.no_mac_randomization:
             logger.info("Changing {} MAC address to {}".format(ap_iface, rogue_ap_mac))
             print "[{0}+{1}] Changing {2} MAC addr (BSSID) to {3}".format(G, W, ap_iface, rogue_ap_mac)
-
-            if not self.advanced_enabled():
+            if self.advanced_enabled():
```

Comment:
The logic here looks incorrect. Monitor adaptor is enabled when `advanced_enabled()` is true.